.github/workflows/continuousIntegration.yaml

```diff
@@ -53,10 +53,13 @@ jobs:
         DHL_ECOMMERCE_SOLUTIONS_CLIENT_SECRET: ${{ secrets.DHL_ECOMMERCE_SOLUTIONS_CLIENT_SECRET }}
         DHL_ECOMMERCE_SOLUTIONS_ENVIRONMENT_URL: ${{ secrets.DHL_ECOMMERCE_SOLUTIONS_ENVIRONMENT_URL }}
         FEDEX_ACCOUNT_NUMBER: ${{ secrets.FEDEX_ACCOUNT_NUMBER }}
```

Comment:
I don't see the point in leaving FEDEX_ACCOUNT_NUMBER here. `account_number` is still set in the Bloodhound constructor but doesn't appear to be used. Same goes for `FEDEX_ENVIRONMENT` and `FEDEX_KEY` and `FEDEX_METER_NUMBER` and `FEDEX_PASSWORD`

Reply:
100%. Those are not needed any longer and I'll remove them.